apps/frontend/src/pages/projects/components/project-card/styles.module.css

```diff
@@ -1,12 +1,16 @@
 .project {
 	display: flex;
 	align-items: center;
+	justify-content: space-between;
 	height: 81px;
 	padding: 0 24px;
```

Comment:
It's better to use `min-height` instead of fixed `height`. And in Figma padding is `16px 24px`, so adjust this too please